app/models/experiment.rb

```diff
@@ -433,7 +432,7 @@ def create_result_csv(with_index=false, with_params=true, with_moes=true, error_
       csv << header
       query_fields = {_id: 0}
       query_fields[:index] = 1 if with_index
-      query_fields[:values] = 1 if with_params
+      query_fields[:values] = 1; query_fields[:input_parameters] = 1  if with_params
```

Comment:
Racja. Zmiana ideologicznie odnosi sie do brancha o input parameters. Prawdopodobnie z uwagi na prace nad 2 branchach rownolegle w bazie mialem jakies śmieci  i ten fragment nie chciał mi działać. Poprawie 1 linie na 2 bo prawdopodobnie jest zle.
Zmiana jest potrzebna bo filtrowania parametrów inaczej wytnie input parameters. To jest wlasnie problem z filtrowaniem parametrów.